add module logistic_requisition_department

lepistone opened the pull request:
dependencies:
- [x] OCA/department#27

lepistone committed:
add logistic_requisition_department

lepistone committed:
add department to lrl view

lepistone committed:
add department to lrs view

lepistone committed:
propagate department from lrs to tender

lepistone committed:
travis: pull OCA/department

yvaucher commented:
:+1:

nbessi commented on logistic_requisition_department/model/logistic_requisition.py:
```diff
@@ -0,0 +1,52 @@
+    _inherit = 'logistic.requisition'
+
+    def _get_my_department(self):
+        my_user = self.env['res.users'].browse(self.env.uid)
```
self.env.user will be better

lepistone replied:
fixed

lepistone committed:
use self.env.user

nbessi commented on logistic_requisition_department/tests/test_propagate_department.py:
```diff
@@ -0,0 +1,20 @@
+        self.lrs = self.env['logistic.requisition.source'].new({
+            'name': '/',
+        })
+        self.dep_rd = self.browse_ref('hr.dep_rd')
```
I'll prefer  `self.env.ref('hr.dep_rd')` matter of taste non blocking

nbessi commented:
:+1: I'll merge once green

lepistone committed:
use self.env.ref

I prefer it because it works everywhere, not only on tests. It makes
sense to use the env for that.

lepistone commented:
@nbessi I applied your suggestions, I agree (I didn't know about the `user` and `ref` properties of `Environment`). Thanks
